numpy/linalg/linalg.py

```diff
@@ -1963,6 +1969,11 @@ def lstsq(a, b, rcond=-1):
                                  0, work, lwork, iwork, 0)
         lwork = int(work[0])
         liwork = iwork[0]
+        if any(size == 0 for size in (lwork, liwork)):
+            raise RuntimeError(
+                'LAPACK did not compute workspace sizes: len(work) = {}, len(iwork) = {}'
+                .format(lwork, liwork)
+            )
```

Comment:
This is a known and fixed (9 Jun 2010) bug in LAPACK: http://icl.cs.utk.edu/lapack-forum/archives/lapack/msg00899.html/. The `scipy.linalg` code has a very similar test, that decides to just error out if this bug is present in LAPACK.

This condition is not hit on my machine (Windows, Intel MKL).

However, this fails on Appveyor. What version of LAPACK is being used there, and do we really need to support a version released before python 2.7.0?